VCFv4.3.tex

```diff
@@ -503,41 +503,100 @@ \subsection{VCF tag naming conventions}
+\indent CHR2 = chromosome of 2nd breakpoint\\
+\indent END = coordinate of 2nd breakpoint\\
+\indent STRAND = (head-tail, tail-head, head-head, tail-tail)\\
+\indent REF = reference base at position CHROM:POS\\
+\indent ALT = reference base at position CHR2:END\vspace{0.3cm}\\
+STR (Short tandem repeat variation, simple sequence repeat, microsatellite): Short tandem repeat variants may be represented in VCF either as structural variants or as ordinary indels (see Section 5.3 for examples). The \textbf{Pindel} software package, for example, treats STRs as standard variants, while \textbf{lobSTR} treats them as structural variants. When treated as structural variants:\vspace{0.3cm}\\
+\indent RPA = repeats per allele\\
+\indent END = end coordinate of the STR\\
+\indent MOTIF = the canonicalized STR repeat motif\\
+\indent NS = number of samples with data\\
+\indent REF = the reference copy number\\
+\indent RL = total length of the STR tract in the reference\\
+\indent RU = repeat motif as it appears on the forward strand in the reference\vspace{0.3cm}\\
+\indent More information on representing STRs in VCF can be found at http://lobstr.teamerlich.org/file-formats.html\vspace{0.3cm}\\
+INV (Inversion):  Describing a canonical inversion is straightforward: it is the excision of a sequence followed by its reinsertion at the same location in reverse orientation. However, actual biological inversions are almost always more complex, involving loss or gain of material at breakpoints, and often co-occurring with secondary SV events such as deletions or duplications. See SVTYPE OTHER for more information.\\
+\\
+DUP (Duplication):  A region of elevated copy number relative to the reference. Depending on the methods by which a given duplication was ascertained, its location and orientation in relation to the reference may or may not be known. If it was detected by read depth analysis or microarray, the location of the duplicate copy in the non-reference genome is probably unknown. The following SVTYPEs are preferred over DUP because they provide more information:\vspace{0.3cm}\\
+\indent TANDUP (Tandem duplication): A duplication located immediately adjacent to the original copy. The position and orientation of the duplicate, if known, should be specified with the TDLOC tag.\vspace{0.3cm}\\
+\indent DISDUP (Dispersed duplication): A duplication located somewhere other than immediately adjacent to the original copy.\\
 \footnotesize
 \begin{verbatim}
 ##INFO=<ID=SVLEN,Number=.,Type=Integer,Description="Difference in length between REF and ALT alleles">
 \end{verbatim}
 \normalsize
-One value for each ALT allele. Longer ALT alleles (e.g. insertions) have positive values, shorter ALT alleles (e.g. deletions) have negative values.
+One value for each ALT allele. ALT alleles that are longer than the reference (e.g. insertions) have positive values, while those shorter than the reference (e.g. deletions) have negative values.
```

Comment:
We should say something about what to write for BND records? No SVLEN defined? Write SVLEN if phased as part of a complex variant?